geode-core/src/main/java/org/apache/geode/security/AuthTokenEnabledComponents.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.security;
+
+public enum AuthTokenEnabledComponents {
```

Comment:
Instead of adding this new external enum, should we instead add MANAGEMENT to SecurableCommunicationChannels? Would the channels already defined by possible auth-token channels in the future? Why do we call them channels sometimes and other times call them components?

Reply:
no, SecurableCommunicationChannels are "ssl" related stuff, it's orthogonal to auth tokens.

Reply:
as to why do we call them channels or components, I believe the config properties are named "components", that's what I am going by. "channels" are just some implementation classes.